lib/ClangImporter/ClangImporter.cpp

```diff
@@ -580,7 +580,8 @@ void importer::getNormalInvocationArguments(
     }
   }
 
-  if (LangOpts.hasFeature(Feature::SafeInteropWrappers))
+  if (LangOpts.hasFeature(Feature::SafeInteropWrappers) &&
```

Comment:
I saw some crashes in Clang when this option was used in C++ mode.

Reply:
`-fbounds-safety` support for C++ in Clang is currently experimental, but `-fexperimental-bounds-safety-attributes` is used for C/C++ interop also, so that one shouldn't crash. It is a relatively new flag though, so there might be some gaps. CC @patrykstefanski for awareness.